rest-api/src/main/resources/paths/uploads/v3_uploads_uploadId_complete.yml

```diff
@@ -13,9 +18,17 @@ post:
         - BridgeSecurity: []
     parameters:
         - $ref: ../../index.yml#/parameters/uploadId
+        - name: synchronous
+          description: True if this API call should wait for upload validation to finish before returning.
+          default: false
+          required: false
```

Comment:
I think required is handled by listing this under a required: property outside of parameters, and you can remove it. Then again if it doesn't fail the build, ok to leave it.